app/controllers/products_controller.rb

```diff
@@ -1,23 +1,23 @@
 class ProductsController < ApplicationController
   def new
-    @product = Product.new 
+    @product = Product.new
   end
 
-  def show 
+  def show
     @product = Product.find(params[:id])
   end
 
-  def create 
+  def create
     @product = Product.new(product_params)
```

Comment:
Can anyone add products or only the admins?

Reply:
already modified it :D